main.js

```diff
@@ -1,13 +1,12 @@
 //  OpenShift sample Node application
 var express = require('express'),
-    bodyParser = require("body-parser"),
-    app     = express();
-    
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
It seems from below that express itself provides what body parser does(Basically the json body parsing)?
Verified?

Reply:
Yes using Express >= 4.16.0, body parser has been re-added under the methods express.json() and express.urlencoded().
http://expressjs.com/en/api.html